### PR TITLE
Removed domain, existing vhost

### PR DESCRIPTION
**Pull request by kazet**

### CODE REVIEW

File: artemis/modules/removed_domain_existing_vhost.py [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+                    continue
+
+                ratio = SequenceMatcher(
+                    None, response_for_old_domain.content, response_for_other_vhost.content
+                ).quick_ratio()
+                self.log.info(f"Similarity between correct and incorrect domain is {ratio} nd status code is {response_for_old_domain.status_code}")
+
+                if (
+                    ratio < Config.Modules.RemovedDomainExistingVhost.REMOVED_DOMAIN_EXISTING_VHOST_SIMILARITY_THRESHOLD
+                    and response_for_old_domain.status_code in range(200, 300)
```
Comment (bulkowy) [inline artemis/modules/removed_domain_existing_vhost.py:112]: I would flip these two conditions

File: artemis/modules/removed_domain_existing_vhost.py [hunk 2]
```diff
@@ -0,0 +1,140 @@
...
+        self.db.save_task_result(
+            task=current_task,
+            status=TaskStatus.OK,
+            status_reason=f"Didn't detect that any of ips: {target_ips} host {domain}",
+        )
+
+
+if __name__ == "__main__":
+    if (
+        Config.Modules.RemovedDomainExistingVhost.REMOVED_DOMAIN_EXISTING_VHOST_PASSIVEDNS_URL
```
Comment (bulkowy) [inline artemis/modules/removed_domain_existing_vhost.py:136]: I guess, similarity should be checked as well

File: artemis/reporting/modules/removed_domain_existing_vhost/translations/en_US/LC_MESSAGES/messages.po [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+#: artemis/reporting/modules/removed_domain_existing_vhost/template_removed_domain_existing_vhost.jinja2:17
+msgid ""
+"If the domain has been removed, we also recommend removing the "
+"corresponding virtual host on the web server. Currently, an attacker "
+"could communicate with the server directly and still interact with the "
+"site, even though it's considered removed and may not be updated."
+msgstr ""
+
+#~ msgid ""
```
Comment (bulkowy) [inline artemis/reporting/modules/removed_domain_existing_vhost/translations/en_US/LC_MESSAGES/messages.po]: to remove old translations

File: artemis/modules/removed_domain_existing_vhost.py (hunk 2 above)
Comment (kazet) [reply artemis/modules/removed_domain_existing_vhost.py:136]: similarity is provided as default, for configs with defaults we don't do such check

**Review by bulkowy — DISMISSED** [review]

**Pull request merged**